vms/platformvm/vm.go

```diff
@@ -64,10 +64,6 @@ type VM struct {
 	*network.Network
 	validators.State
 
-	common.Subscriber
-
-	closeSubscriber func()
```

Comment:
how do we fail fast on the context once we shut down? 

Maybe we should wire the parent context `vm.onShutdownCtx` to all our calls?

Reply:
I would have expected the consensus engine to have cancelled the call to the `WaitForEvent` prior to calling `Shutdown`